zstacklib/zstacklib/utils/sanlock.py

```
@@ -1,20 +1,36 @@
-from zstacklib.utils import log, linux, thread
-
+import os.path
 import re
 import random
+import struct
+import traceback
 from string import whitespace
+
+from zstacklib.utils import log, sizeunit
```

Ruff (0.11.9)

`zstacklib.utils.sizeunit` imported but unused

Remove unused import: `zstacklib.utils.sizeunit`

(F401)